variables.tf

```diff
@@ -55,4 +55,9 @@ variable "name_suffix" {
 variable "filter_str_concatenation" {
   description = "If you use an IN expression you need to switch from , to AND"
   default     = ","
-}
\ No newline at end of file
+}
+
+variable "priority_offset" {
+  description = "For non production workloads we can +1 on the priorities"
```

Comment:
For alerts this is how it works:
```
priority = default priority + priority_offset
```

So for example if you have a development environment you set `priority_offset = 1` so that the priority is "lower" 

<img width="448" alt="image" src="https://user-images.githubusercontent.com/174411/163123397-9044d5c3-816e-4933-8309-12bd51d2f57d.png">